src/main/java/net/coderbot/iris/mixin/MixinWorldRenderer.java

```diff
@@ -232,4 +232,24 @@
 		profiler.swap("iris_pre_translucent");
 		pipeline.beginTranslucents();
 	}
+
+	@Inject(method = RENDER, at = @At(value = "CONSTANT", args = "stringValue=blockentities"), locals = LocalCapture.CAPTURE_FAILHARD)
+	private void iris$StartBlockEntities(MatrixStack matrices, float tickDelta, long limitTime,
+										 boolean renderBlockOutline, Camera camera, GameRenderer gameRenderer,
+										 LightmapTextureManager lightmapTextureManager, Matrix4f matrix4f,
+										 CallbackInfo ci, Profiler profiler, Vec3d vec3d, double d, double e, double f,
+										 Matrix4f matrix4f2, boolean bl, Frustum frustum2, boolean bl3,
+										 VertexConsumerProvider.Immediate immediate) {
+		pipeline.setPhase(WorldRenderingPhase.BLOCK_ENTITIES);
+	}
+
+	@Inject(method = RENDER, at = @At(value = "CONSTANT", args = "stringValue=destroyProgress"), locals = LocalCapture.CAPTURE_FAILHARD)
+	private void iris$EndBlockEntities(MatrixStack matrices, float tickDelta, long limitTime,
+									   boolean renderBlockOutline, Camera camera, GameRenderer gameRenderer,
+									   LightmapTextureManager lightmapTextureManager, Matrix4f matrix4f,
+									   CallbackInfo ci, Profiler profiler, Vec3d vec3d, double d, double e, double f,
+									   Matrix4f matrix4f2, boolean bl, Frustum frustum2, boolean bl3,
+									   VertexConsumerProvider.Immediate immediate) {
+		pipeline.setPhase(WorldRenderingPhase.OTHER);
+	}
```

Comment:
I don't think that local capture is needed here. Also, the convention is `iris$startBlockEntities` afaik.

Reply:
Fixed, would you mind reviewing again